src/Glpi/Application/ConfigurationConstants.php

```diff
@@ -41,8 +41,14 @@ public function __construct(private string $root_dir)
     {
     }
 
-    public function computeConstants(): void
+    public function computeConstants(?string $env = null): void
     {
+        if ($env !== null) {
```

Comment:
```suggestion
        if ($env !== null && !defined('GLPI_ENVIRONMENT_TYPE')) {
```

Allows calling it more than once

Reply:
It should not be called more than Twice. Also, it would silently fail silently and people would not understand why the `GLPI_ENVIRONMENT_TYPE` is not set to the value they are passing to the `Kernel` constructor.